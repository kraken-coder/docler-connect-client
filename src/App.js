import React from 'react'

import { createGlobalStyle } from 'styled-components'
import AppRoute from './components/global/AppRoute'

const App = () => (
  <>
    <GlobalStyle />
    <AppRoute />
  </>
)

// Resets and global styling
const GlobalStyle = createGlobalStyle`
*{
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

body{
  font-family: san-serif
  font-size: 10px;
  position: relative;
}

`
export default App
