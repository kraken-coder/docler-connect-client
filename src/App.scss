@import './template/layout.scss';

.dark {
  background-color: #121212;
  font-family: ' Raleway ', sans-serif;
  font-size: 10px;
  color: color(primary);
  height: 100vh;
}
