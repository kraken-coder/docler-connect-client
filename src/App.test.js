import React from 'react'
import { shallow } from 'enzyme'
import configureStore from 'redux-mock-store'
import toJSON from 'enzyme-to-json'
import App from './App'

// mock state
const initialState = {
  darkTheme: false,
}
const mockStore = configureStore()
const store = mockStore(initialState)

test('match snapshot', () => {
  const wrapper = shallow(<App store={store} settings="dark" />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})
