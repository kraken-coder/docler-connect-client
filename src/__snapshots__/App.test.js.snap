// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<ContextProvider
  value={null}
>
  <App
    dispatch={[Function]}
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
  />
</ContextProvider>
`;
